local/container.go

```diff
@@ -31,11 +31,16 @@ func (c *container) URL() *url.URL {
 	}
 }
 
-func (c *container) CreateItem(name string) (stow.Item, io.WriteCloser, error) {
-	path := filepath.Join(c.path, name)
+func (c *container) pathAndItem(name string) (string, *item) {
```

Comment:
not sure crazy about this method, why not just do it inline?

Reply:
Also, the unexported methods should go after all the exported methods for the `container` type